clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -10425,8 +10425,13 @@ def warn_type_safety_null_pointer_required : Warning<
   "specified %0 type tag requires a null pointer">, InGroup<TypeSafety>;
 
 // Generic selections.
-def err_assoc_type_incomplete : Error<
-  "type %0 in generic association incomplete">;
+def ext_assoc_type_incomplete : Extension<
+  "ISO C requires a complete type in a '_Generic' association; %0 is an "
+  "incomplete type">;
```

Comment:
Any reason why this doesn’t say ‘is incompatible with C standards before C2y’?

Reply:
Or rather, ‘is a C2y extension’

Reply:
Sure, I can change that!